src/SPC/builder/extension/pgsql.php

```diff
@@ -36,16 +36,24 @@ public function patchBeforeConfigure(): bool
     public function getUnixConfigureArg(bool $shared = false): string
     {
         if ($this->builder->getPHPVersionID() >= 80400) {
-            return '--with-pgsql PGSQL_CFLAGS=-I' . BUILD_INCLUDE_PATH . ' PGSQL_LIBS="-L' . BUILD_LIB_PATH . ' -lpq -lpgport -lpgcommon"';
+            $libfiles = $this->getLibFilesString();
+            $libfiles = str_replace(BUILD_LIB_PATH . '/lib', '-l', $libfiles);
+            $libfiles = str_replace('.a', '', $libfiles);
+            $libfiles = str_replace(' -lpq', '', $libfiles);
+            $libfiles = str_replace(' -lpgport', '', $libfiles);
+            $libfiles = str_replace(' -lpgcommon', '', $libfiles);
+            return '--with-pgsql' . ($shared ? '=shared' : '') .
+                ' PGSQL_CFLAGS=-I' . BUILD_INCLUDE_PATH .
+                ' PGSQL_LIBS="-L' . BUILD_LIB_PATH . ' -lpq -lpgport -lpgcommon ' . $libfiles . '"';
```

Comment:
This seems too straightforward. Why can't it accept the value of `getLibFilesString()`?

Reply:
There's something wrong with the way the libs are interpreted. When using static paths, it fails to resolve symbols. I can remove the -lpq -lpgport -lpgcommon stuff though.